app/Helpers/DatabaseHelper.php

```diff
@@ -7,41 +7,41 @@
 use App\Models\GamesFile;
 use Carbon\Carbon;
 
-class DatabaseHelper{
+class DatabaseHelper {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->